portfolio/src/main/webapp/script.js

```diff
@@ -19,10 +19,24 @@ function addRandomGreeting() {
   const greetings =
       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
+
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
+
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomline() {
+    const lines=
+        ['Im a peacock you gotta let me fly','just do it','crewmen','clap your hands if you believe', 'FIRE EVERYTHING!','You will be obliterated']
+  // Pick a random greeting.
```

Comment:
Nit: remove this comment? Seems like it doesn't apply to the code anymore

Reply:
```suggestion
  // Pick a random line.
```
 I changed it to be more specific to the function